### PR TITLE
created catalog page

### PR DESCRIPTION
**Pull request by igorkruz**
- [DEMO LINK](https://igorkruz.github.io/layout_catalog/)
- [TEST REPORT LINK](https://igorkruz.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
-body {
```
Comment (BondarenkoDmitriy) [inline src/styles/main.scss:1]: it would be better to move it to the card and header

**Review by BondarenkoDmitriy — APPROVED** [review]
good job

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
```
Comment (VinnieJ-2k20) [inline src/styles/blocks/card.scss:13]: Do not add margin rules to blocks.

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__img {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
```
Comment (VinnieJ-2k20) [inline src/styles/blocks/header.scss:14]: Move to separate file.

File: src/styles/blocks/page.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.page {
+  &__catalog {
+    padding: 50px 0 0;
+  }
+}
+
+.catalog {
```
Comment (VinnieJ-2k20) [inline src/styles/blocks/page.scss:7]: Move to separate file.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Almost done. Need to fix some styles.

**Comment by igorkruz** [comment]
> Almost done. Need to fix some styles.

fixed

File: src/styles/blocks/page.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.page {
+  &__catalog {
+    padding: 50px 0 0;
```
Comment (VinnieJ-2k20) [inline src/styles/blocks/page.scss:3]: You can add paddings to blocks without BEM mix

**Review by VinnieJ-2k20 — APPROVED** [review]

**Pull request closed without merging**